### PR TITLE
Feature branch

### PR DESCRIPTION
**Pull request by paulzay**
### What does this PR do?
Build a todo app
### Description of tasks to be completed?
1. Build projects functionality logic
2. Build todos functionality logic
3. Display both todos and projects on the dom

### CODE REVIEW

File: src/modules/domStuff.js [hunk 1]
```diff
@@ -0,0 +1,258 @@
+/* eslint-disable import/prefer-default-export */
```
Comment (Forison) [inline src/modules/domStuff.js]: - do not make modification to you eslint configs

File: src/modules/logic.js [hunk 2]
```diff
@@ -0,0 +1,82 @@
+/* eslint-disable import/no-mutable-exports */
```
Comment (Forison) [inline src/modules/logic.js]: do not make modification to you eslint configs

File: package.json [hunk 3]
```diff
@@ -0,0 +1,34 @@
+{
+  "name": "to-do-list",
+  "version": "1.0.0",
+  "description": "As part of Microverse JS curriculum",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
```
Comment (Forison) [inline package.json:10]:
```javascript
"scripts": {
    "dev": "webpack --mode development",
    "build": "webpack --mode production",
    "start": "webpack-dev-server --mode development --open" // optional
  },
```

**Review by Forison — CHANGES_REQUESTED** [review]
## Status:: Changes Required✖️
# 🙋‍♂️Dzhumaliev & Zay
#### Good job done so far, your code is well grouped in modules, here are a few issues we have to address before getting the project approved👨‍🔧.
* [x]  Linters config: Do not make any changes in config files - they represent style guidelines that you share with your team - which is a group of all Microverse students.([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/linter_config.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/54883773/))
* [x] The link to the live demo in the readme is not directing to the actual page, kindly address this issue.
* [x] There should be a 'default' project.
- when the user has not created a project, all todos that are added must be stored under the default project name.
* [x] Necessary run common commands like `build`, `development` should be in `package.json` inside \`scripts'. Here is a basic example ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/todo_list/scripts_requirement.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59622602/))
Kindly include check my comments below

### That is it, kindly make the changes and submit a re-review👨‍🔧.

#### Happy coding 👨‍💻
Addo Forison

**Review by Dandush03 — APPROVED** [review]
# Status: :rocket: Project Approved :rocket: 
### Great Work! Keep It Up :muscle: 
# Happy Coding! :nerd_face: :computer:

**Pull request merged**